### PR TITLE
Utils singleton

### PR DESCRIPTION
**Pull request by akhil-ghatiki**
# Description
Implemented Bill Pugh singleton design pattern for FileStoreUtils.java and FileUriUtils.java. Made changes in corresponding files where these methods are used.

Changed the variable names in FileUriUtils.java as per the check style.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by akhil-ghatiki** [comment]
@Swati4star Please look into this PR.

Also, I think FileStoreUtils and FileUriUtils are not orthogonal. I think we need to look into them to make them loosely coupled.

File: app/src/main/java/swati4star/createpdf/util/FileSortUtils.java [hunk 1]
```diff
@@ -7,12 +7,12 @@
 public class FileSortUtils {
 
     // Sorting order constants
-    public static final int NAME_INDEX = 0;
-    public static final int DATE_INDEX = 1;
-    public static final int SIZE_INCREASING_ORDER_INDEX = 2;
-    public static final int SIZE_DECREASING_ORDER_INDEX = 3;
+    public final int NAME_INDEX = 0;
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/FileSortUtils.java]: shouldnt these be static?

File: app/src/main/java/swati4star/createpdf/util/FileUriUtils.java [hunk 2]
```diff
@@ -177,11 +177,19 @@ static String getImageRealPath(ContentResolver contentResolver, Uri uri, String
...
             return null;
         path =  path.replace("/document/raw:", "");
         return path;
     }
+
+    private static class SingletonHolder {
+        static final FileUriUtils INSTANCE = new FileUriUtils();
+    }
+
+    public static FileUriUtils getInstance() {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/FileUriUtils.java]: make this class's contructor private?

File: app/src/main/java/swati4star/createpdf/util/FileSortUtils.java (hunk 1 above)
Comment (akhil-ghatiki) [reply app/src/main/java/swati4star/createpdf/util/FileSortUtils.java]: As they are class members and we are having a static instance. I thought accessing these variables through static instance is good.

File: app/src/main/java/swati4star/createpdf/util/FileUriUtils.java (hunk 2 above)
Comment (akhil-ghatiki) [reply app/src/main/java/swati4star/createpdf/util/FileUriUtils.java]: Will make it and push the changes again.

**Comment by Swati4star** [comment]
Great! merging this.
and I agree with

> Also, I think FileStoreUtils and FileUriUtils are not orthogonal. I think we need to look into them to make them loosely coupled.

You can work on above next if you want?

**Pull request merged**